packages/extension/src/utils/format.ts

```diff
@@ -121,6 +127,15 @@ const LABEL_SET_NO_RIPPLE = 'Set No Ripple';
 const LABEL_CLEAR_NO_RIPPLE = 'Clear No Ripple';
 const LABEL_SET_FREEZE = 'Set Freeze';
 const LABEL_CLEAR_FREEZE = 'Clear Freeze';
+// AMMDeposit
+const LABEL_LPTOKEN = 'LP Token';
```

Comment:
This is already in another file, maybe we should have only one source of truth

Reply:
It's not the same purpose ! transaction flag VS Token listing